Example/Authenticator/Utils/Extensions/SEEncryptedDataExtensions.swift

```diff
@@ -37,7 +37,7 @@ extension SEBaseEncryptedAuthorizationData {
         guard let v2Response = self as? SEEncryptedAuthorizationData,
               let connectionId = connectionId else { return nil }
 
-        if v2Response.status.isFinal {
+        if v2Response.status.isFinal || v2Response.status.isClosed {
```

Comment:
Not needed. As soon as we don't need to see `closed` authorizations - we don't need to map it to model.